src/components/SubmitResponsePage.jsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import Alert from './Alert';
+import Alert from '../utilities/Alert';
```

Comment:
Alert is actually a component and should be reanmed to `jsx` that is my bad on that one.

Reply:
Moved back to jsx